readme.md

```diff
@@ -1424,13 +1424,20 @@ your `vite.config.js`.
 
 #### WMR
 
-~~[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
-[pushing them to `config.plugins`](https://github.com/preactjs/wmr/tree/main/packages/wmr#configuration-and-plugins)
-in `wmr.config.js`.~~
+[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
+[adding them to `plugins`](https://wmr.dev/docs/plugins)
+in `wmr.config.mjs`.
 
-WMR *says* it supports Rollup plugin but apparently it fails on `.mdx` files
-before the Rollup plugin.
-See [GH-62](https://github.com/wooorm/xdm/issues/62).
+```
+import { defineConfig } from 'wmr';
+import xdm from 'xdm/rollup.js';
+
+export default defineConfig({
+   plugins: [
+       xdm({ /* Options... */ })
+   ]
+});
+```
```

Comment:
I figure an example couldn't hurt.

Now, I am a bit torn on the XDM options, so I'm wondering what others think. WMR is totally UI library agnostic, users can use other UI libraries with it or none at all.

That being said, the default new project template does ship with Preact, so if users were to use that template without alterations, they'd need to add `jsxImportSource: 'preact'` to the options object.

Not sure if that should be in this little snippet or not.

Reply:
I missed this comment. Yeah you had `preact` in the comment in the other issue, right?

Maybe add a link to how to integrate with Preact?

`See [§ Preact](https://github.com/wooorm/xdm#preact) if you want to use Preact.`

Reply:
I’m fine with `/* Options */`. I think that implies you need to go look at what options you can pass. `{jsxImportSource: 'preact', /* Options */ }` slightly implies that Preact is needed?

Reply:
I’m fine with what it currently is too

Reply:
Yeah I'm probably a bit sensitive to this as people already tend to assume that since it's from the Preact org it must only work with Preact, but that's not the case.

As of right now we only provide a Preact template for initializing new projects, and while that may or may not change in the future, I totally get gearing instructions towards that template, even if it's open to be customized by end users to say use React instead of Preact.

Adding a footnote to go look at the Preact instructions is probably the best middle ground but I'm totally fine with catering to the new project template as well. I believe most users of WMR will be using Preact, and most will probably be creating new projects using that template rather than adding the lib to an already initialized app.

Reply:
Whatever you think will result in xdm getting less questions/issues, don't want to bother you with our confused users 😅

Reply:
Alright, then let’s go with the note from https://github.com/wooorm/xdm/pull/69#discussion_r670193144 or so?

Under the example?

Reply:
Sorry for the delay, busy day. Just added that note in.